main.go

```diff
@@ -78,12 +99,12 @@ func (m *MozProcessStat) Add(data MozProcessStat) {
 	m.Memory.Swap += data.Memory.Swap
 	// Available memory must not be summed.
 
-	m.CPU.User += data.CPU.User
-	m.CPU.System += data.CPU.System
-	m.CPU.Idle += data.CPU.Idle
-	m.CPU.Iowait += data.CPU.Iowait
-	m.CPU.Steal += data.CPU.Steal
-	m.CPU.Percent += data.CPU.Percent
+	// m.CPU.User += data.CPU.User
```

Comment:
Nit: Given that we now no longer add these, maybe we could remove them altogether?